Example/ViewController.swift

```diff
@@ -258,11 +287,16 @@ class ViewController: UIViewController {
 
         customViewController.userIndexedRoute = (route, 0)
         customViewController.userRouteOptions = routeOptions
-
-        // TODO: Add the ability to show destination annotation.
         customViewController.simulateLocation = simulationButton.isSelected
+        
+        passiveLocationDataSource?.systemLocationManager.stopUpdatingLocation()
```

Comment:
In my opinion this is pretty significant limitation: we have to stop any free-drive related location updates to prevent clash with locations sent during active guidance. At the same time there is no easy way to do this under the hood, because of public nature of `PassiveLocationDataSource`. Any better ideas on how we can improve this?